babel.config.js

```diff
@@ -4,17 +4,22 @@ const React = require('react');
 const errorCodesPath = path.resolve(__dirname, './docs/public/static/error-codes.json');
 const missingError = process.env.MUI_EXTRACT_ERROR_CODES === 'true' ? 'write' : 'annotate';
 
+function resolveAliasPath(relativeToBabelConf) {
+  const resolvedPath = path.relative(process.cwd(), path.resolve(__dirname, relativeToBabelConf));
+  return `./${resolvedPath.replace('\\', '/')}`;
```

Comment:
Out of curiosity, why not use an absolute path?

Reply:
I would prefer an absolute path as well but I recall this being problematic. You would transpile  files to 

```js
import Typography from '/home/eps1lon/Development/material-ui/packages/material-ui/src/Typography';
```

And I think bundlers/nextjs would then treat them as external files and not transpile the imported files. Though it's very likely that this was just a symptom of another problem (that might've been fixed). 

I can check this in a follow-up (want to see a full,clean CI/CD run) to make sure. 

Technically, we should be able to transpile it to `import Typography from 'file://home/eps1lon/Development/material-ui/packages/material-ui/src/Typography'` i.e. use an actual URI which is valid in ES6 modules. Though this would require a lot of other infra work.

Reply:
I think the biggest problem is that this would result in `C:\\users\eps1lon\eps1lon\Development\material-ui\packages\material-ui\src\Typography` which might be considered a "remote" file (definitely by `rsync`). This may not be an issue but we get into really weird territory where it's hard to reason about "where" the file is. 

I'm not even 100% sure if win32 paths are valid in ES6 modules. So this is all "just works" code that has no solid foundation.

Reply:
Your explanation makes a lot of sense. I recall trying this change too and seeing warnings raised by rollup.

Reply:
Though definitely right to call this out. This code has now more visibility and the next person who runs into issues with that code is aware that this might just be completely wrong code.